exporter/views.py

```diff
@@ -35,13 +37,81 @@ def get_original_asset_id(download_url):
     """
     if download_url.startswith("http://tile.loc.gov/"):
         pattern = r"/service:([A-Za-z0-9:\-]*)/"
```

Comment:
At the very least our service identifiers should be at least one character long, right?

```suggestion
        pattern = r"/service:([A-Za-z0-9:\-]+)/"
```